README.md

```diff
@@ -1159,6 +1159,20 @@ Vagrant.configure(2) do |config|
 end
 ```
 
+## Custom command line arguments
+You can also specify multiple commandline arguments for qemu-system
+
+* `value` - Value
+
+```ruby
+Vagrant.configure("2") do |config|
+  config.vm.provider :libvirt do |libvirt|
+    libvirt.commandline :value => "-device"
+    libvirt.commandline :value => "isa-applesmc,osk=ourhardworkbythesewordsguardedpleasedontsteal(c)AppleComputerInc"
+  end
+end
+```
+
```

Comment:
I believe you're missing an addition to the merge method in the `VagrantPlugins::ProviderLibvirt::Confg` class to ensure that additional blocks will merge the `commandline` settings together.

Haven't check it myself, but I'd expect the following to have unexpected behaviour
```
Vagrant.configure("2") do |config|
  config.vm.provider :libvirt do |libvirt|
    libvirt.commandline :value => "-device"
    libvirt.commandline :value => "isa-applesmc,osk=ourhardworkbythesewordsguardedpleasedontsteal(c)AppleComputerInc"
  end

  # this block will override the previous block for commandline
  config.vm.provider :libvirt do |libvirt|
    libvirt.commandline :value => "-device"
    libvirt.commandline :value => "dummy-device"
  end
end
```

If you want subsequent blocks to always override previous within the same Vagrantfile, this is fine, if you want blocks to append, not so much.